command/v7/push_command.go

```diff
@@ -363,11 +371,73 @@ func (cmd PushCommand) GetFlagOverrides() (v7pushaction.FlagOverrides, error) {
 		NoManifest:          cmd.NoManifest,
 		Task:                cmd.Task,
 		LogRateLimit:        cmd.LogRateLimit,
+		Lifecycle:           cmd.Lifecycle,
 	}, nil
 }
 
 func (cmd PushCommand) ValidateFlags() error {
 	switch {
+	// Lifecycle buildpack requested
+
+	case cmd.Lifecycle == constant.AppLifecycleTypeBuildpack && cmd.DockerImage.Path != "":
```

Comment:
Since we are checking for dockerImage and docker-username, would it make sense to also test for docker-password?

Reply:
Wrong input combinations, in regards to docker, are checked earlier. I guess it does not make sense to check it again.